dockerfiles/examples/nfs/values-storage-nfs.yaml

```diff
@@ -0,0 +1,15 @@
+# Overrides for azure storage in hpcc, using custom StorageClass'
+# NB: The "retained-storage" helm charts should be installed 1st.
```

Comment:
Should "retained-storage" say "nfs-server here? Should the existingClaim names reflect the storage type in use?